### DISCUSSION TITLE
FSRS-5

### DISCUSSION THREAD

**Discussion by joshdavham** [open-spaced-repetition/free-spaced-repetition-scheduler#25] [Q&A] [answered]
I've noticed that work seems to be underway for updating the various FSRS implementations to v5.

At first glance it seems like the changes are the following:
1. Updated parameters
     - FSRS v5 has now has 19 weights vs 17 weights
 2. New initial difficulty
     - Found [here](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm#fsrs-5-draft)
 3. New stability formula for short term memory
     - Also found [here](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm#fsrs-5-draft)

Are there other changes I'm missing? And do we have the greenlight to start updating the various FSRS implementation to V5?

**Comment by L-M-Sherlock** [comment]
In my opinion, the greenlight is the next Anki release.

**Reply by joshdavham** [reply]
Yeah that sounds reasonable.

Question before I mark as answer: are the changes I've listed above correct? And what is meant by short-term memory in that new stability formula?

**Reply by L-M-Sherlock** [reply]
https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm#fsrs-5-draft

The short-term memory formula is applied when the user reviews the same card in the same day.

**Reply by joshdavham** [reply]
Hmm I'm not sure if I totally understand. Could I get an example?

Suppose I have a card that I made on Day 1 and I rate it as 'Good' each time it comes up, until say Day 100 where there's a lapse and I fail it by rating 'Again'. Because the card will now show up again on Day 100, does it get the formula applied to it then?

Or rather does it simply mean that, if a given card has already been reviewed within the last 24 hours, that the next stability calculation will be the short term one? If it's this one, do new cards that haven't been reviewed yet have the short term formula applied to them?

**Reply by L-M-Sherlock** [reply]
For example, you need to rate 'Good` twice for a new card in the same day in Anki to enter in the long-term review. So the first rating is used to initialize the initial stability, and the second rating is used to update the memory state via short-term formula.

**Reply by joshdavham** [reply]
Okay I think that makes sense. But just to confirm, in your example, if that card were to 'lapse' in the future, it would have that short term formula applied again?

**Reply by L-M-Sherlock** [reply, marked as answer]
If a card were to 'lapse' and had one or more extra relearning stages in the same day, the short-term formula is applied again.

**Reply by joshdavham** [reply]
Alrighty that completely makes sense!
